index.Rmd

```diff
@@ -83,15 +83,15 @@ summary(fit)
 Show tables, plots, etc. and describe them.
 
 ```{r echo=TRUE, warning=FALSE}
-ggplotly (g, tooltip = c("County", "Opioid.Poisoning.Deaths", "Year"))
+ggplotly (g, tooltip = c("County", "Opioid.Poisoning.Deaths", "Year")) 
     
 ```
 
 ```{r echo=TRUE}
-mapview(vital_ny_2017, zcol = "Opioid.Poisoning.Deaths", legend = TRUE, alpha = 0.5)
+mapview(vital_ny_2017, zcol = "Opioid.Poisoning.Deaths", legend = TRUE, alpha = 0.5) 
 ```
```

Comment:
Good job! This graph gives me an intuitive feel about the death situation. But it could be better if you could give us a more detailed description about the story of this graph.
Because I noticed that change the color distribution is unbalanced in your graph, or say the classification is kind of rough.  And I could see that the case number in each county are more likely in the range 0-50 and just few counties have very high numbers.  So I am curious about why those counties have such high numbers of cases.